utils/src/main/java/com/cloud/utils/fsm/StateMachine2.java

```diff
@@ -50,7 +50,11 @@ public void addInitialTransition(E event, S toState) {
     }
 
     public void addTransition(S currentState, E event, S toState) {
-      addTransition(new Transition<S, E>(currentState, event, toState, null));
+      addTransition(currentState, event, toState, false);
+    }
+
+    public void addTransition(S currentState, E event, S toState, boolean alertFlag) {
+      addTransition(new Transition<S, E>(currentState, event, toState, null, alertFlag));
     }
```

Comment:
Old constructor just calls new constructor with False for the alertFlag.  So we don't break any functionality using the previous constructor